examples/spring-boot/marketplace-app/README.md

```diff
@@ -0,0 +1,107 @@
+
+# Marketplace App Demo
+
+Repository with sample code for building marketplace-app. It will help partners to kickstart their own app using Vendasta platform.
+
+## Getting Started
+
+- [Installation](#installation)
+- [API Endpoints](#api-endpoints)
+- [Used Tech Pluggins](#used-tech-pluggins)
+- [Running Tests](#running-tests)
+- [Related Links](#related-links)
+
+
+### Installation
+
+Clone the project
+
+```bash
+  git clone https://github.com/vendasta/api-gateway-docs
+```
+
+Go to the project directory
+
+```bash
+  cd examples/spring-boot/marketplace-app
+```
+
+* Inside the demo code, [ngrok](https://ngrok.com) is used to make a temporary domain, but you are free to replace it with your own.
```

Comment:
We should probably tell them to create the app in vendor center at this point. We can give them all the values that they would need to fill into the SSO form. Then if they don't have a domain already we can suggest ngrok. (It's too bad we block localhost.)